src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -954,6 +956,42 @@ void TouchedMethodsDCmd::execute(DCmdSource source, TRAPS) {
   VMThread::execute(&dumper);
 }
 
+ClassesDCmd::ClassesDCmd(outputStream* output, bool heap) :
+                                     DCmdWithParser(output, heap),
+  _verbose("-verbose",
+           "Dump the detail content of Java class. "
+           "Some classes are annotated with flags: "
+           "F = has finializer method, "
```

Comment:
typo finializer - but should be finalize

Is this actually only present for "non-trivial finalize" method?

Reply:
I'm not sure what's the meaning of "non-trivial finalize" method, can you elaborate more for it?

Reply:
I mean a finalize() method that actually does something. I checked the code and you will print F is the current class has a non-empty finalize() method, or it has a superclass with a non-empty finalize method. I would suggest updating the text to:

`F = has, or inherits, a non-empty finalize method`

Thanks,
David

Reply:
Done. This description is more clear compared to non-trivial finalize.